src/js/sort.js

```diff
@@ -13,6 +13,12 @@ const SORT_OPTIONS = {
   freq_cn: '中国語での使用頻度',
 };
 
+const SORT_OPTIONS_SHORT = {
+  stroke: '画',
```

Comment:
Hmmm....

On PC, we can put the long-forms into... Tool-tips or little messages which pop-up on hover or something, but I wonder what we should do on mobile? Since there's no hover...